packages/shared/src/games.ts

```diff
@@ -69,4 +70,5 @@ export type Score = {
   details: readonly ScoreDetail[];
   total: number;
   totalRound: number;
+  winner: boolean;
```

Comment:
I think this should either:
a) be the top of the state, as `winner` and value being the player name, instead of carrying it for every player
b) not be part of the state, let the client figure it out. Although we might need to store it in the room, to keep track of wins

Reply:
I changed this to be winning player, it's not the absolute winner but it's a flag that says if a player is winning or not. We can change it later, let's see if it works this way